plugins/tpm/fu-self-test.c

```diff
@@ -189,9 +189,9 @@ static void
 fu_tpm_empty_pcr_func(void)
 {
 	gboolean ret;
-	g_autofree gchar **environ = NULL;
 	g_autofree gchar *pluginfn = NULL;
 	g_autofree gchar *testdatadir = NULL;
+	g_auto(GStrv) environ_old = NULL;
```

Comment:
I'm fine with the change but I'm curious why clang thinks there is a clash: is the global `environ` from libc?

Also is there a typo in the commit message? I think you mean it's *not* a `gchar**`, is it?